src/h3lib/include/h3api.h.in

```diff
@@ -260,6 +275,16 @@ double H3_EXPORT(hexAreaKm2)(int res);
 
 /** @brief average hexagon area in square meters (excludes pentagons) */
 double H3_EXPORT(hexAreaM2)(int res);
+
+/** @brief exact area for a specific cell (hexagon or pentagon) in radians^2 */
+double H3_EXPORT(cellAreaRads2)(H3Index h);
```

Comment:
I think we may need a new `@defgroup` here? As I understand it:
- Each `@defgroup` defines a named function the bindings should expose
- When the @defgroup doesn't match the function names, it's generally because the bindings are expected to expose only one function using several public functions in C. The main case here is the combo of a max size function and a function that takes allocated memory - bindings expose `polyfill` and use `maxPolyfillSize` under the hood.
- I think for `hexArea` we've used this to allow bindings to have one function that takes units as a param (that's what `h3-js` does). But `cellArea` is a different function entirely, even if we offer `cellArea(h3, unit)`, so it needs its own group.

Reply:
done!